### PR TITLE
Portuguese Help Message

### PR DESCRIPTION
**Pull request by hugok79**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

File: lang/pt/cmd_help [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    list        Listar pacotes com base nos nomes dos pacotes
+    log         Mostrar registos
+    purge       Purgar pacotes
+    run         Executar um comando dentro do container
+    remove      Remover pacotes
+    search      Pesquisar nas descrições dos pacotes
+    show        Mostrar detalhes dos pacotes
+    update      Atualizar lista de pacotes disponíveis
+    upgrade     Atualizar o sistema ao instalar/atualizar pacotes
+    version     Mostrar versão e sair
```
Comment (kbdharun) [inline lang/pt/cmd_help:24]:
```suggestion
```

Comment (hugok79) [reply lang/pt/cmd_help]: > 

Done

Comment (kbdharun) [reply lang/pt/cmd_help]: > > 
> 
> Done!

Thanks

**Comment by hugok79** [comment]
Latest commit: a7c47b5c3ff4f7f2a45b47cf697ed86eb61bf55a included

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution @hugok79. Sorry for the delay, finally we have a well-defined system for managing translations, if possible can you create a new/edit the current PR adding a `pt.yml` file under the locales directory? (You can copy and then rename the `en.yml` file)

Our current system unifies man pages and cmd help translations in a single place and translations.

**Comment by hugok79** [comment]
> Thanks for your contribution @hugok79. Sorry for the delay, finally we have a well-defined system for managing translations, ~if possible can you create a new/edit the current PR adding a `pt.yml` file under the locales directory? (You can copy and then rename the `en.yml` file to your language code.)~
> 
> ~Our current system unifies man pages and cmd help translations in a single place.~
> 
> Edit: Apx translations are now available in weblate, feel free to contribute here https://hosted.weblate.org/projects/vanilla-os/apx/pt_BR/

Thanks for the info. Altough I only translate  Portuguese is https://hosted.weblate.org/projects/vanilla-os/apx/pt/ ! not Brazilian Portuguese https://hosted.weblate.org/projects/vanilla-os/apx/pt_BR/

**Pull request closed without merging**